Misc/NEWS.d/next/Documentation/2024-01-17-11-40-03.gh-issue-114123.LuueXf.rst

```diff
@@ -0,0 +1,5 @@
+Moved `csv` module docstring from `Modules/_csv.c` to `Lib/csv.py` and
```

Comment:
`Modules/_csv.c` and `Lib/csv.py` are not meaningful names for common Python user. It is better to write about `csv` and `_csv` modules.

Also not that it is reStructuredText, not Markdown. Use ``` :mod:`csv` ``` if you want to create a reference and ``` :mod:`!csv` ``` or just ``` ``csv`` ``` if do not.